Reworks Preternis into being distinct from IPCs and augmented humans

Moltijoe opened the pull request:
Really early stage preternis rework, this is **REALLY** far from done, numbers aren't a factor yet, those can be tweaked once all the mechanical changes are done.
The main idea is to change them from just being IPC2: empkilled boogaloo, into something more unique and closer to the lore that Skrem has written, focused heavily on being sorta insectoid robotic workers and not specialized for combat.
Started the pull now so people can weigh in on the ideas and perhaps give me pointers on how to make things work neatly.

### Primary tasks
- [x] Remove innate slowdown (let's be real, no one wants to be slower than default)
- [x] Complete Emp immunity
- [x] Water weakness (ideally a similar scale or greater weakness as the emp weakness)
- [x] Give them the innate ability to maglock
- [x] Maglocking icon
- [x] Make them faster with tools 
- [x] Make em slow at doing the hurty hurt
- [x] Balance
- [x] Make them take more temperature damage but resist temperature changes
- [x] Get more community feedback

### **This needs to be merged before a test merge can happen**
https://github.com/yogstation13/Yogstation/pull/16165

### Secondary tasks (might get own PR)
- [x] [Make it easier to tell to what degree you have been soaked
](https://github.com/yogstation13/Yogstation/pull/15892)
- [x] [Turn the maglock ability into two separate implants, maglock implant, and antislip implant ](https://github.com/yogstation13/Yogstation/pull/15864)

### Coming in a future PR
Give them TRAIT_DIGITIGRADE and make sure there are no issues with that (won't even need footwraps because of metal limbs)
Redone Preternis Sprites making them look more insectoid


Make water vapor make you wet (janitor getting ahelped increase by 1000%)

general fire_stacks tweaks


Adds an icon debuff for being wet
![Screenshot_1](https://user-images.githubusercontent.com/108117184/190840010-4b8ae260-4fdc-4c1d-a9f0-20dc192c4326.png)

:cl:  
rscadd: Added new Preternis
rscdel: Removed old Preternis
imageadd: added an alert icon  
/:cl:

Big thanks to both Theos and Mqiib for help with various coding things
And Skrem for the design direction to go

Moltijoe committed:
Preternis rework 1

EMP immunity
Water fuckery
Digitigrade legs
Tall n Skinny

Moltijoe committed:
Adds icon to wetness debuff
start attempt at adding the maglock feet ability (my brain go ouchie)

Moltijoe committed:
Preternis now have an action speed increase, might be too strong, not sure what all it applies to

Moltijoe committed:
Reworks action speed
Finishes maglock
Removes Digitigrade (still no shoes though)
Adds poor aim (might need a rework to aim debuffs)
Initial attempts at slowing down melee speed

Moltijoe committed:
sets up maglock icon
adds proper wetness examine

Moltijoe committed:
Stat tweaks
Change noslipall to noslipwater
Slows down hurting people

Moltijoe committed:
Tweaks the lungs to be more suited for hot temperatures

Moltijoe committed:
Mucks with some power things

Moltijoe committed:
Merge branch 'yogstation13:master' into PreternisRework

Moltijoe committed:
Ignore that last commit

Moltijoe committed:
snowflake blood

Moltijoe committed:
Removes digitigrade again, will be its own PR along with resprite

Moltijoe committed:
Little bit more number balancing

Moltijoe committed:
saw moths had a brute mod of 0.9

Moltijoe committed:
Removes attack slowdown if has martial art

Moltijoe committed:
Makes firestack code not aids to read

Moltijoe committed:
Reverts the fire_stacks examine text

Moltijoe committed:
Variable water damage
Small splashes hurt less than before, large ones hurt more

Skrem7 reviewed (commented):
minor syntax lookover

Moltijoe committed:
Update yogstation/code/modules/mob/living/carbon/human/species_types/preternis/organs.dm

Lungs description change

Co-authored-by: Skrem_7 <98909416+Skrem7@users.noreply.github.com>

Moltijoe committed:
Update yogstation/code/modules/mob/living/carbon/human/species_types/preternis/screen_alerts.dm

Alert exclamation point

Co-authored-by: Skrem_7 <98909416+Skrem7@users.noreply.github.com>

Moltijoe committed:
Makes preternis heat up and cool down more slowly, but once they get too hot or too cold they take way more damage

Moltijoe committed:
Conflict resolve?

Moltijoe committed:
fixing conflict

Moltijoe committed:
conflict fix

Moltijoe committed:
nothing's working?

Moltijoe committed:
Test?

Moltijoe committed:
dunno

Moltijoe committed:
Merge pull request #1 from yogstation13/master

Update rework

Moltijoe committed:
Okay, should be fully updated now

Moltijoe committed:
Makes the maglock ability consume more power from them
Increases max power ever so slightly
Makes the hunger alert pop up only at 50% rather than closer to 65%

Moltijoe committed:
nevermind, changing powerdrain just doesn't work for some reason

Moltijoe committed:
Merge branch 'master' into PreternisRework

Moltijoe committed:
Updates the magboots ability to not be quite as jank

Moltijoe committed:
Merge branch 'PreternisRework' of https://github.com/Moltijoe/Yogstation into PreternisRework

Moltijoe committed:
makes empproof actually get removed when swapping species

Moltijoe committed:
Couple tweaks that should be a better starting point for a testmerge

Moltijoe committed:
protect self, not contents

Moltijoe committed:
Voracious scales the speed of power drain

Moltijoe committed:
Couple more stat tweaks

Moltijoe committed:
Return reduced health limbs
reverts unintended one-tap stun batons (oopsie)
gives very minor DR to body

Moltijoe committed:
couple more tweaks
